apps/widget/cypress/e2e/notifications-list.spec.ts

```diff
@@ -85,4 +85,12 @@ describe('Notifications List', function () {
     cy.wait('@seenRequest');
     cy.getByTestId('unseen-count-label').contains('4');
   });
+
+  it('count seen-unseen notification', function () {
+    cy.getByTestId('unseen-count-label').contains('5');
+    cy.intercept('**/messages/**/seen').as('seenRequest');
+    cy.getByTestId('notification-list-item').first().click();
+    cy.wait('@seenRequest');
+    cy.getByTestId('notification-list-item').should('have.length', 5);
+  });
```

Comment:
Maybe worth waiting for the notification list request after the click too? Because maybe it would be faster than the list of feed request here. 

Also, suggest adding another API e2e test for the behaviour.